portfolio/src/main/webapp/script.js

```diff
@@ -25,26 +25,69 @@ function changeTextBack(text) {
   display.innerHTML = text;
 }
 
+//Form Text Input Resizing Function - Name
+let elName = document.querySelector(".input-wrap-name .input");
+let extraWidthName = document.querySelector(".input-wrap-name .extra-width-name");
+elName.addEventListener("keyup", () => {
+  extraWidthName.innerHTML = elName.value;
+});
+
+//Form Text Input Resizing Function - Message
+let elMessage = document.querySelector(".input-wrap-message .input");
+let extraWidthMessage = document.querySelector('.input-wrap-message .extra-width-message');
+elMessage.addEventListener("keyup", () => {
+  extraWidthMessage.innerHTML = elMessage.value;
+});
+
 function loadComments() {
-  fetch('/list-comments').then(response => response.json()).then((comments) => {
+  var numComments = document.getElementById("num-comments").value;
+
+  fetch('/list-comments?num=' + numComments).then(response => response.json()).then((comments) => {
     const commentListElement = document.getElementById('comments-list');
+    document.getElementById('comments-list').innerHTML = "";
     comments.forEach((comment) => {
-      commentListElement.appendChild(createCommentElement(comment));
+      const commentElement = document.createElement('p');
+      commentElement.className = 'comment-element';
+      commentElement.appendChild(createDiv(comment.author, 'text-css-label'));
+      commentElement.appendChild(createDiv(comment.message, 'text-css-value'));
+      commentElement.appendChild(createDiv('}', ''));
+      commentListElement.appendChild(commentElement);
     })
   });
 }
 
-function createCommentElement(comment) {
-  const commentElement = document.createElement('div');
-  commentElement.className = 'comment-element';
+function createDiv(passMessage, passClass) {
```

Comment:
What does "passMessage" (and "passClass") represent? Are you referring to the idea that it's a parameter that's being passed?

Reply:
Yes, the passed message and passed class